projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.js

```diff
@@ -0,0 +1,22 @@
+( function () {
+	var windowObjectReference = null;
+	function init() {
+		document.addEventListener( 'click', function ( e ) {
+			if ( e.target.dataset.fauxInlineHelp !== undefined ) {
+				e.preventDefault();
+
+				if ( windowObjectReference == null || windowObjectReference.closed ) {
+					windowObjectReference = window.open( e.target.href, e.target.target );
+				} else {
+					windowObjectReference.focus();
+				}
+			}
+		} );
+	}
+
+	if ( document.readyState === 'loading' ) {
+		document.addEventListener( 'DOMContentLoaded', init );
+	} else {
+		init();
+	}
```

Comment:
Is this needed given that the `inline-help.js` script is loaded in the footer?

Reply:
Not strictly. But I've been bitten enough times by someone switching the boolean on `enqueue_script` and then finding errors in my JS code.

Reply:
Also see this is the pattern used elsewhere in `masterbar` module.